ts/Core/Renderer/BBoxObject.d.ts

```diff
@@ -24,6 +24,7 @@ import type SizeObject from './SizeObject';
  * */
 
 export interface BBoxObject extends PositionObject, SizeObject {
+    centerX?: number;
```

Comment:
I wanted to add this property but backed out from it, kept this property anyway, because I've seen, that it is used here: https://github.com/highcharts/highcharts/blob/34d9e7d7d07186a9da34b18b7c358000472f2501/ts/Core/Series/DataLabel.ts#L198

I can delete it, if you think so.

Reply:
Oh, I see. Please delete it and add a `// @todo` in DataLabel to create a custom dlBox type there.